docs/en/stack/ml/anomaly-detection/ootb-ml-jobs-apm.asciidoc

```diff
@@ -80,15 +80,17 @@ This job is useful in identifying bots.
 [[apm-transaction-jobs]]
 == Transactions
 // tag::apm-transaction-jobs[]
-Detect anomalies in transactions from your APM services. These configurations 
-are only available if data exists that matches the following term query: 
-`"processor.event": "transaction"`.
+Detect anomalies in transactions from your APM services for metric data. These 
+configurations are only available if data exists that matches the following term 
+queries: `"processor.event": "metric"` and `"metrcset.name": "transaction"`.
```

Comment:
Typo - should be `metricset.name`, and with the code styling, I don't think you need the quotes `"` around the field names and values here.

Or alternatively, do we want to follow the text used elsewhere `These configurations are only available if data exists that matches the recognizer query specified in the manifest file.` and leave the details for where you describe the job below? The link for the manifest won't be available until https://github.com/elastic/kibana/pull/111957 is merged, but would be `https://github.com/elastic/kibana/blob/master/x-pack/plugins/ml/server/models/data_recognizer/modules/apm_transaction/manifest.json#L8`

Reply:
Changed in c348ee5.